app/domain/authentication/authn_gcp/decoded_token.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
```

Comment:
How are these 3 new files fixing merge conflicts? Also, shouldn't these be integrated into the functional commits in which the conflict was introduced?

Reply:
They don't. I rearranged the commits now so that we don't have this commit anymore